cmd/mocks/client.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by mockery v2.10.4. DO NOT EDIT.
+// Code generated by mockery v2.10.6. DO NOT EDIT.
```

Comment:
How do we generate this?

Reply:
Basically `mockery --all` is executed in the directory where the interface resides. In this case, ANR client directory. The file is then renamed and moved to destiny. Args of mockery can be used to avoid renaming. A script in ANR does the automatic generation for mocks that ANR needs. Maybe we can add also this kind of script to CLI

Reply:
https://ava-labs.atlassian.net/browse/AV-1842

Reply:
perfect, I was modding those files by hand, glad to know.